### DISCUSSION TITLE
Feature: Shortcut/widget to a specific note on mobile

### DISCUSSION THREAD

**Discussion by diegogurpegui** [standardnotes/forum#3325] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
No, just a nice to have.

**Describe the solution you'd like**
I would like the ability to create shortcut in the mobile home (in the form of a widget) that opens a specific note directly. This could be very handy for those notes that one is constantly opening, like a checklist, a note where you dump ideas, etc.

**Describe alternatives you've considered**
The SN app should allow for the creation of a widget that asks you to select a note. Then a new icon in the home appears and you can put it wherever you like. The name of the icon can be chosen by the user too, since the note data should/could be encrypted at rest.
There's a special consideration when you have the app locked, with fingerprints for example. What happens there? Well, when you click on the icon it asks for your fingerprint before opening the note. Or, a more complex solution, could be to allow non-protected notes.

**Additional context**
This was thought with Android in mind. I've seen similar things on other apps, for example "Google Keep", "Contacts", etc. where you can have a widget icon that points to one specific element.

**Comment by FractalHabits** [comment]
This would be amazing

**Comment by Kidlike** [comment]
This would be massively helpful also for the "capture" stage of the GTD (getting things done) methodology.

A simple shortcut opening a specific note, ideally at the end of it (or beginning of it, whatever is easier), with the keyboard already open, would be ideal.

This almost completely removes all friction that currently is there for "capturing" (finding the app, opening the app, finding the note, opening the note, opening the keyboard), which can take even up to 10 seconds, where it could be <1.

Then, the answer to "Is your feature request related to a problem?" becomes a "Yes" -- @diegogurpegui feel free to update your answer :innocent:
